ykrt/build.rs

```diff
@@ -1,3 +1,5 @@
 pub fn main() {
     ykbuild::apply_llvm_ld_library_path();
+
+    println!("cargo:rustc-cfg=jitc_llvm");
```

Comment:
Should this be a cargo feature instead?

Reply:
It's not obvious to me that there any advantages to one over t'other. Any suggestions?

Reply:
A cargo feature can be configured without source code changes.